src/__tests__/mapping-entry.test.ts

```diff
@@ -6,56 +6,30 @@ describe("mapping-entry", () => {
     const result = getAbsoluteMappingEntries(
       "/absolute/base/url",
       {
-        "*": ["/foo1", "/foo2"],
-        "longest/pre/fix/*": ["/foo2/bar"],
+        "*": ["/foo1", "./foo2"],
+        "longest/pre/fix/*": ["./foo2/bar"],
         "pre/fix/*": ["/foo3"],
       },
       true
     );
-    // assert.deepEqual(result, [
-    //   {
-    //     pattern: "longest/pre/fix/*",
-    //     paths: [join("/absolute", "base", "url", "foo2", "bar")],
-    //   },
-    //   {
-    //     pattern: "pre/fix/*",
-    //     paths: [join("/absolute", "base", "url", "foo3")],
-    //   },
-    //   {
-    //     pattern: "*",
-    //     paths: [
-    //       join("/absolute", "base", "url", "foo1"),
-    //       join("/absolute", "base", "url", "foo2"),
-    //     ],
-    //   },
-    // ]);
     expect(result).toEqual([
       {
         pattern: "longest/pre/fix/*",
         paths: [join("/absolute", "base", "url", "foo2", "bar")],
       },
       {
         pattern: "pre/fix/*",
-        paths: [join("/absolute", "base", "url", "foo3")],
+        paths: [join("/foo3")],
```

Comment:
I believe these test cases were broken by https://github.com/dividab/tsconfig-paths/pull/184. I updated them to have a sampling of both relative and absolute paths.